gradoop-flink/src/main/java/org/gradoop/flink/model/api/epgm/GraphCollectionOperators.java

```diff
@@ -255,7 +255,7 @@ GraphCollection apply(
 
   /**
    * Transforms a graph collection into a logical graph by applying a
-   * {@link BinaryGraphToGraphOperator} pairwise on the elements of the
+   * {@link BinaryBaseGraphToBaseGraphOperator} pairwise on the elements of the
```

Comment:
Guess this was changed by automatic refactoring. Either way, both are confusing.  Set the link to `ReducibleBinaryGraphToGraphOperator.`